dev/integration_tests/flutter_gallery/test/demo/material/buttons_demo_test.dart

```diff
@@ -35,9 +35,9 @@ void main() {
     {
       await tester.tap(find.text('OUTLINED'));
       await tester.pumpAndSettle();
-      expect(find.byType(OutlinedButton).evaluate().length, 2);
+      expect(find.byType(OutlinedButton).evaluate().length, 4);
```

Comment:
Changed this test as originally OutlinedButton.icon was returning _OutlinedButtonWithIcon, and  didnt count as an OutlinedButton.